front/src/pages/PrivatePages/hooks/useGet.jsx

```diff
@@ -3,166 +3,173 @@ import { useContext, useState } from 'react';
 import { useNavigate } from 'react-router-dom';
 import Context from '../../../context/Context';
 
-
-function useGet(){
-
-    const url=process.env.REACT_APP_API_URL;
-    const baseUrl =`${url}/admin`;
-    const token=localStorage.getItem('token');
-    const { logOut } = useContext(Context);
-
-    const [studentList,setStudentList]=useState([]);
-    const [eventList,setEventsList]=useState([]);
-    const [adviserList,setAdviserList]=useState([]);
-    const [studentDetail, setStudentDetail] = useState();
-    const [totalPagesEvent,seTotalPagesEvent]=useState(0);
-    const [totalEventsGet,setTotalEventsGet]=useState();
-    const [totalEvents,setTotalEvents]=useState();
-
-    const navigate = useNavigate();
-
-    const LogOut = () => {
-        logOut();
-        navigate('/login', { replace: true })
+function useGet() {
+  const url = process.env.REACT_APP_API_URL;
+  const baseUrl = `${url}/admin`;
+  const token = localStorage.getItem('token');
+
+  const { logOut } = useContext(Context);
+
+  const [studentList, setStudentList] = useState([]);
+  const [eventList, setEventsList] = useState([]);
+  const [adviserList, setAdviserList] = useState([]);
+  const [studentDetail, setStudentDetail] = useState();
+  const [totalPagesEvent, seTotalPagesEvent] = useState(0);
+  const [totalEventsGet, setTotalEventsGet] = useState();
+  const [totalEvents, setTotalEvents] = useState();
+  const [adviserDetail, setAdviserDetail] = useState();
+
+  const navigate = useNavigate();
+
+  const LogOut = () => {
+    logOut();
+    navigate('/login', { replace: true });
+  };
+
+  const options = {
+    method: 'GET',
+    headers: {
+      'Content-Type': 'multipart/form-data',
+      'x-token': `Bearer ${token}`,
+    },
+  };
+
+  const calculateTotalPages = (totalEvent) => {
+    let totalPages = 0;
+    let restEvents = 0;
+    let total = totalEvent;
+    while (total > 10) {
+      if (total > 10) {
+        totalPages += 1;
+      }
+      total -= 10;
+      restEvents = totalPages;
     }
-
-    const options= {
-        method: 'GET',
-        headers: { 'Content-Type': 'multipart/form-data'
-        ,'x-token':`Bearer ${token}`},
-    };
-
-    const calculateTotalPages = (totalEvent)=>{
-        let totalPages=0
-        let restEvents=0
-        let total=totalEvent;
-        while(total>10){
-          if(total>10){
-            totalPages+=1
-          }
-          total-=10;
-          restEvents=totalPages;
-        }
-        if(restEvents===0 && total>0){
-          totalPages += 1;
-        }
-        else if(restEvents>0){
-          totalPages += 1;
-        }
-        seTotalPagesEvent(totalPages);
+    if (restEvents === 0 && total > 0) {
```

Comment:
Como el if y el elseif tienen lineas que hacen lo mismo, podrias juntar los ifs a uno solo.
```suggestion
    if ((restEvents === 0 && total > 0) || restEvents > 0) {
```